lever.lua

```diff
@@ -0,0 +1,209 @@
+-- Inspect and pull levers
+world = df.global.world
```

Comment:
what does this gain from being global?

Reply:
I cargoculted it from somewhere else, I think 🤔